plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/index.tsx

```diff
@@ -18,8 +18,13 @@ import Tooltip from './Tooltip'
 
 import * as styles from './styles.module.css'
 import { TogglesContext, TogglesProvider } from './ToggleProvider'
+import { linkIcon } from '../../../../../../static/icons'
+import SVG from 'react-inlinesvg'
```

Comment:
Instead of adding an extra package, could we use `dangerouslySetInnerHTML` to insert the SVG string?

Reply:
Yes, thanks.